features/telemetry/protocols/bmp/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/bmp/parser/proto/bgp/packets/Capability.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2020 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2020 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.netmgt.telemetry.protocols.bmp.parser.proto.bgp.packets;
+
+import static org.opennms.netmgt.telemetry.listeners.utils.BufferUtils.slice;
+import static org.opennms.netmgt.telemetry.listeners.utils.BufferUtils.uint8;
+
+import com.google.protobuf.ByteString;
+
+import io.netty.buffer.ByteBuf;
+
+public class Capability {
+
+    private final int code;
+    private final int length;
+    private final ByteBuf value;
+
+    public Capability(final ByteBuf buffer) {
+        this.code = uint8(buffer);
+        this.length = uint8(buffer);
+        this.value = slice(buffer, this.length);
+    }
+
+    public int getCode() {
+        return code;
+    }
+
+    public int getLength() {
+        return length;
+    }
+
+    public ByteString getValue() {
+        return ByteString.copyFrom(value.array());
```

Comment:
Not that it currently is required to implement right now, but do we have a clue about what this value could be?

Reply:
We currently implement the parsing for this value for code 1 and 69, like the OpenBMP implementation does.